lib/elixir/lib/exception.ex

```diff
@@ -640,8 +640,26 @@ defmodule UndefinedFunctionError do
   end
 
   def message(%{reason: :"module could not be loaded", module: module, function: function, arity: arity}) do
+    suffix =
+      case :code.get_object_code(module) do
```

Comment:
This will probably be cleaner if you use the `with` construct. Also keep in mind we can pass the second argument from `:code.get_object_code` to `:beam_lib`. Here is one suggestion:

```elixir
suffix =
  with {_, binary, _} <- :code.get_object_code(module),
       {:ok, {alt_module, _}} <- :beam_lib.chunks(binary, []) do
    exports = exports_for(alt_module)
    if ... do ... else ... end
  end
```